samples/subsys/usb/cdc_acm/prj.conf

```diff
@@ -1,21 +1,20 @@
 CONFIG_STDOUT_CONSOLE=y
-CONFIG_SYS_LOG=y
 CONFIG_GPIO=y
 CONFIG_USB=y
 CONFIG_USB_DEVICE_STACK=y
 CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM sample"
 CONFIG_LOG=y
 CONFIG_USB_DRIVER_LOG_LEVEL=1
 CONFIG_USB_CDC_ACM=y
-CONFIG_SYS_LOG_USB_DEVICE_LEVEL=1
+CONFIG_USB_DEVICE_LOG_LEVEL=1
```

Comment:
Unfortunately this does not work anymore:
```
warning: USB_DEVICE_LOG_LEVEL (defined at subsys/logging/Kconfig.template.log_config:22) was             
assigned the value '1' but got the value '3'. This symbol has no prompt, meaning assignments in          
configuration files have no effect on it.
```
two options:
```diff
--- a/samples/subsys/usb/cdc_acm/prj.conf                                                               
+++ b/samples/subsys/usb/cdc_acm/prj.conf                                                               
@@ -4,9 +4,9 @@ CONFIG_USB=y                                                                            
 CONFIG_USB_DEVICE_STACK=y                                                                              
 CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM sample"                                                      
 CONFIG_LOG=y                                                                                           
-CONFIG_USB_DRIVER_LOG_LEVEL=1                                                                          
+CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y                                                                      
 CONFIG_USB_CDC_ACM=y                                                                                   
-CONFIG_USB_DEVICE_LOG_LEVEL=1                                                                          
+CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y                                                                      
 CONFIG_SERIAL=y                                                                                        
 CONFIG_UART_INTERRUPT_DRIVEN=y                                                                         
 CONFIG_UART_LINE_CTRL=y                                                                                
@@ -14,7 +14,7 @@ CONFIG_UART_LINE_CTRL=y                                                               
 CONFIG_USB_COMPOSITE_DEVICE=n                                                                          
                                                                                                        
 CONFIG_USB_MASS_STORAGE=n                                                                              
-CONFIG_USB_MASS_STORAGE_LOG_LEVEL=1                                                                    
+CONFIG_USB_MASS_STORAGE_LOG_LEVEL_ERR=y    
```

or
```diff
--- a/subsys/logging/Kconfig.template.log_config
+++ b/subsys/logging/Kconfig.template.log_config
@@ -20,7 +20,7 @@ config $(module)_LOG_LEVEL_DBG
 endchoice
 
 config $(module)_LOG_LEVEL
-       int
+       int "Log level"
        default 0 if $(module)_LOG_LEVEL_OFF
        default 1 if $(module)_LOG_LEVEL_ERR
        default 2 if $(module)_LOG_LEVEL_WRN

```

I am for the second solution.

Reply:
I noticed the same thing in networking so one needs to add the _DBG etc to the end of the log level variable. In networking I opted to use the first method as if you add `int "Log level"` then the numeric level is also shown to user in menuconfig. So there would be two settings for the same thing, which could imho be quite confusing.

Reply:
I used the first option to avoid confusion.